docs/events/add-manage-notes.asciidoc

```diff
@@ -0,0 +1,64 @@
+[[add-manage-notes]]
+= Notes
+
+Incorporate notes into your investigative workflows to coordinate responses, conduct threat hunting, and share investigative findings. You can attach notes to alerts, events, and Timeline and manage them from the **Notes** page. 
+
+NOTE: Configure the `securitySolution:maxUnassociatedNotes` <<max-notes-alerts-events,advanced setting>> to specify the maximum number of notes that you can attach to alerts and events. 
+
+[discrete]
+[[add-notes-documents]]
+== Add notes to alerts and events
```

Comment:
something still feels off to me here. I feel like we're repeating ourselves too many times.

What do you think about renaming this section `View and add notes to alerts and events`?

Then the content could be something like
`Open the alert or event details flyout to access the **Notes** tab, where you can view the notes already created for that alert or event, and add new notes. The summary in the right panel shows how many notes are attached to the alert.`

_Insert an updated screenshot that shows a note already created and a note being created_

`To quickly open the tab, use the **Add note** action (image:images/create-note-icon.png[Add note action,15,15]) in the Alerts or Events table. Then, enter a note into the text box, and click **Add note** to create it. Once notes are created, the **Add note** icon will display a notification red dot over it.`

_Insert screenshot that show the red dot_

Reply:
Combining the sections sounds good, but I do think the steps for adding notes should be placed first in the title and section body. (Notes need to be created before they can be viewed.)

Reply:
Added in [b438eae](https://github.com/elastic/security-docs/pull/6006/commits/b438eaef9668518300148f12d0c09d0d1b353c10).